google/cloud/bigquery/client.py

```diff
@@ -596,20 +617,35 @@ def create_table(self, table, exists_ok=False, retry=DEFAULT_RETRY, timeout=None
         data = table.to_api_repr()
         try:
             span_attributes = {"path": path, "dataset_id": dataset_id}
-            with create_span(
-                name="BigQuery.createTable", attributes=span_attributes, client=self
-            ):
-                api_response = self._call_api(
-                    retry, method="POST", path=path, data=data, timeout=timeout
-                )
+            api_response = self._call_api(
+                retry,
+                name="BigQuery.createTable",
+                span_attributes=span_attributes,
+                client=self,
+                method="POST",
+                path=path,
+                data=data,
+                timeout=timeout,
+            )
             return Table.from_api_repr(api_response)
         except google.api_core.exceptions.Conflict:
             if not exists_ok:
                 raise
             return self.get_table(table.reference, retry=retry)
 
-    def _call_api(self, retry, **kwargs):
-        call = functools.partial(self._connection.api_request, **kwargs)
+    def _call_api(
+        self,
+        retry,
+        name=None,
```

Comment:
Should this parameter be called `span_name`? Since it's only used to create the span, the parameter should have name that makes its purpose more clear.

Reply:
+1 on calling this `span_name`, since it isn't used to actually make the API call